CHANGELOG.md

```diff
@@ -19,11 +19,72 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
-## @meteora-ag/dlmm [1.3.8] - PR #144
+## @meteora-ag/dlmm [1.3.9] - PR #140
+
+### Changed
+
+#### Breaking
+
+- `getPairPubkeyIfExists` added new `baseFeePowerFactor` parameter
+- `getAllPresetParameters` now return `presetParameter` and `presetParameter2` accounts instead of only `presetParameter`
+- Rename `LBCLMM_PROGRAM_IDS` to `DLMM_PROGRAM_IDS`
+- `createPermissionLbPair` added new `protocolFeeBps` parameter
```

Comment:
do we need this function in sdk? caused it is only used by admin

Reply:
Fixed